### PR TITLE
Bring ros2_control test launches to canopen_tests pkg

### PR DESCRIPTION
**Pull request by ipa-cmh**
As all other packages, test and example launch and configurations should be stored in canopen_tests.
This PR does this for canopen_system and cia402_system example launches.

### CODE REVIEW

File: canopen_tests/CMakeLists.txt [hunk 1]
```diff
@@ -11,6 +11,8 @@ find_package(lely_core_libraries REQUIRED)
 
 
 generate_dcf(simple)
+generate_dcf(canopen_system)
```
Comment (destogl) [inline canopen_tests/CMakeLists.txt:14]: Should we add 'ros2_control' in the name - to make clearer what 'systems' means?

File: canopen_tests/config/canopen_system/bus.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+options:
```
Comment (destogl) [inline canopen_tests/config/canopen_system/bus.yml:1]: I assume you added this to make test independable?

**Review by destogl — APPROVED** [review]
Maybe we don't have to duplicate can configuration files. Then we have to always cage everything multiple times...

File: canopen_tests/config/canopen_system/bus.yml (hunk 2 above)
Comment (ipa-cmh) [reply canopen_tests/config/canopen_system/bus.yml:1]: Yes, we had changed the bus.yaml for cia402 and it broke cia402_system launch. So I'd rather keep things seperate.

File: canopen_tests/CMakeLists.txt (hunk 1 above)
Comment (ipa-cmh) [reply canopen_tests/CMakeLists.txt:14]: We are working on a major documentation update. I think we will add a documentation on example launches.
In that case it does not matter much I suppose.

**Pull request merged**